content/en/developers/marketplace/api_integration.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Create an API Integration
+type: documentation
+further_reading:
+- link: "https://partners.datadoghq.com/"
+  tag: "Partner Network"
+  text: "Datadog Partner Network"
+- link: "/developers/integrations/marketplace"
+  tag: "Documentation"
+  text: "Learn about the Datadog Marketplace"
+- link: "/developers/integrations/oauth_for_integrations"
+  tag: "Documentation"
+  text: "Learn about using OAuth for integrations"
+---
+
+## Overview
+
+You can use [Datadog API endpoints][1] to programmatically send data to Datadog, manage your account, and visualize data collected from an API integration. 
+
+API integrations send the following types of data to Datadog:
+
+- [Metrics][2]
+- [Logs & Log Pipelines][3]
+- [Events][4]
+- [Service Checks][5]
+- [Traces][6]
+- [Incidents][7]
+- [Security Events][8]
+
+This page provides instructions for creating an API integration in the `marketplace` repository. For more information about why you would want to create an API-based integration, see [Creating your own solution][9].
+
+## Setup
+
+### Prerequisites
+
+- You must have an [API key][10] and [application key][11].
+- You know which [Datadog site][12] you want to use.
+
+An API key is required to submit data to a Datadog API endpoint. An application key is required to query data from Datadog or to create resources within the Datadog site. For more information, see [API and Application Key][11].
+
+Create a connection to Datadog in your company's platform using the API key, application key, and site URL, that allows users to configure and install the integration outside of Datadog.
```

Comment:
The Marketplace team would like to require all API based integrations to use OAuth moving forward so this likely doesn't apply. We can replace it with the OAuth section below.

Reply:
```suggestion
```